base/set.jl

```diff
@@ -191,6 +191,37 @@ function unique(f::Callable, C)
 end
 
 """
+	unique!(itr)
+
+Remove duplicate items and return the modified collection,
+as determined by [`isequal`](@ref).
+
+```jldoctest
+julia> unique!([1, 1, 1])
+1-element Array{Int64,1}:
+ 1
+
+julia>  unique!([7, 3, 2, 3, 7, 5])
```

Comment:
I think a better example would be:
```julia
julia>  A = [7, 3, 2, 3, 7, 5];

julia> unique!(A);

julia> A
4-element Array{Int64,1}:
 7
 3
 2
 5
```

To show that it is changing `A`. Otherwise this example would look exactly like `unique` behaves